content/themes/glimmer/assets/css/screen.css

```diff
@@ -1943,8 +1943,9 @@ a:hover, a:active {
 #header-middle #site-logo {
   float: left;
   width: 29.5%;
-  margin-left: 25px;
+  margin-left: 38px;

```

Comment:
ugly hack to make the logo appear centered instead of spending time to do it right and make it pixel perfect